testkit/tests/system_api.rs

```diff
@@ -40,3 +42,14 @@ fn test_user_agent_info() {
     let expected = user_agent::get();
     assert_eq!(info, expected);
 }
+
+#[test]
+fn test_network() {
+    let testkit = TestKitBuilder::validator().with_validators(2).create();
+    let api = testkit.api();
+    let info: NodeInfo = api.get_private(ApiKind::System, "/v1/network");
+
+    assert_eq!(info.core_version, option_env!("CARGO_PKG_VERSION").map(|ver| ver.to_owned()));
```

Comment:
Late-to-the-party comment: As far as I understand, this assumes that the testkit version is equal to the core version, which doesn't *always* hold. I'd check `assert!(info.core_version.is_some())` instead.